src/many_forms_hello_world.md

```diff
@@ -79,7 +79,7 @@ This example demonstrates printing a variable instead of an inline constant.
 
 ### `hello_world_6.ln`
 
-```rust,editable
+```alan,editable
```

Comment:
Will these work right? You have the ACE editor believing that it's Rust?

Reply:
ah you are right, I messed up my find and replace. one sec

Reply:
Well, I would like to approve this change, but we also need to alter ACE's syntax highlighting to know what it's actually highlighting.